python/tests/test_ibis.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import textwrap
+
+import cudf
+import cupy
+import ibis
+
+from legate_dataframe import LogicalTable
+from legate_dataframe.experimental_ibis import LegateBackend
+from legate_dataframe.testing import assert_frame_equal
+
+
+def test_ibis_basic_read_csv_and_binary(tmp_path):
+    file = tmp_path / "test.csv"
+    file.write_text(
+        textwrap.dedent(
+            """\
+        a,b,c
+        1,2,3
+        2,3,4
+        """
+        )
+    )
+
+    schema = ibis.schema({"a": "int64", "b": "float64", "c": "int64"})
+    b = LegateBackend().connect()
+    # NOTE: As of writing, require schema.  In principle should infer schema
+    # and not actually read the full file immediately (i.e. make lazy).
+    table = b.read_csv(file, schema=schema)
+
+    sum_ab = (table.a + table.b).name("sum_ab")
+
+    res = sum_ab.execute()  # A columnar (pandas) result.
+    assert res.name == "sum_ab"  # pandas series has a name and we honor it
+    # Rename res, just for `assert_frame_equal`.
+    assert_frame_equal(res.rename("data"), cudf.Series([3.0, 5.0]))
```

Comment:
Does legate guarantee result order?  For something this small it's unlikely to drift, but if ordering isn't broadly guaranteed, it's good to have that in mind when writing tests (and adding explicit `order_by`s to expected results)

Reply:
Yes, order is stable legate (and except for some manual operations, that is even pretty deep if we use the legate data model).